scripts/tasks.js

```diff
@@ -0,0 +1,19 @@
+window.addEventListener("DOMContentLoaded", function () {
+    const meTooButton = document.getElementById("me_too_button");
+    const meTooMainDiv = document.getElementById("MeTooMain");
```

Comment:
very minor, but the casing of the ids are inconsistent. I recognize MeTooMain was first so maybe we align on that style? Really doesn't matter.

Reply:
OTOH snake case seems generally more common in php files.